### DISCUSSION TITLE
Lista de Exercícios

### DISCUSSION THREAD

**Discussion by giscus** [dunossauro/exercicios-python-brasil#24] [General]
# Lista de Exercícios

http://localhost:8000/

**Comment by heit0r** [comment]
Aparentemente, a Python Brasil retirou os exercícios do ar, espero que temporariamente. Então, uma página como esta se torna ainda mais importante. Belo trabalho ao fazer rodar o código na própria página, parabéns aos envolvidos.

**Comment by JesusFabian113** [comment]
q

**Comment by Benjamim12** [comment]
Gostei muito dor exrcícios

**Comment by lidiajager** [comment]
Que excelente ter tantos exercícios pra terino!
